apps/web/src/components/utils/GotAQuestionButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { useIntercom } from 'react-use-intercom';
+
+import { INTERCOM_APP_ID } from '../../config';
+import { Button, Size } from '../../design-system';
+
+export function GotAQuestionButton({ mt, size }: { mt: number; size: Size }) {
+  if (!INTERCOM_APP_ID) {
+    return null;
+  }
+
+  const { show } = useIntercom();
```

Comment:
A little question here, isn't it advised to use the hooks before any loops or conditional checks? <https://reactjs.org/docs/hooks-rules.html>

Reply:
On a second look, it seem the `InercomProvider` is being conditional added to the app, so I guess this would be only logical.
However if using the `userIntercom` hook alone won't throw any error, I believe it would still be best to not render the hook conditionally?

Reply:
hmm good question. maybe we can create a custom hook and just mock the `show` method conditional?

Reply:
The answer depends if this hook `useIntercom` will be used elsewhere as well.

If yes, then the idea of creating custom hook and mocking is a good idea.
If nope, then we can do the same thing here only.

Just to be safe and future proof though, I would say creating custom hook wouldn't hurt at all. 🙏

Reply:
I was scratching my head about this why Eslint hasn't complained about it and realised that the plugin to flag hooks not implementing them properly wasn't set. I will fix that too in this PR and will take the recommendation.
Good spot @TheLearneer!

Reply:
I have to leave the hooks rules plugin config for a different PR as it is breaking `/apps/web` because we have problems with other hooks implemented.